server/templates/place.html

```diff
@@ -60,7 +65,7 @@ <h3 id="locale" data-lc="{{ locale }}"></h3>
         </div>
       </div>
       <div id="subtitle" class="row"></div>
-      <div id="place-summary" class="row-col place-summary-container">{{ place_summary or '' }}</div>
+      <div id="place-summary" class="row-col place-summary-container">{{ place_summary if locale == "en" }}</div>
```

Comment:
Nit: can you move the locale check to `place/html.py`? i think eventually that's where the place summary localization logic will live, so might make sense to move the "en" check there now. wdyt?

Reply:
Good idea. Moved to html.py